Description: Use Debian Javascript packages.
 * still need to package the following:
   - BootSideMenu.js
   - clipboard.js
   - nouislider.js
 * 3d visualizations aren't working with current three.js
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-06

--- a/M2/Macaulay2/packages/Visualize/digraph-example.html
+++ b/M2/Macaulay2/packages/Visualize/digraph-example.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Digraph Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -59,11 +59,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visDigraph2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 
--- a/M2/Macaulay2/packages/Visualize/graph-example.html
+++ b/M2/Macaulay2/packages/Visualize/graph-example.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Graph Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -84,11 +84,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visGraph2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
       
--- a/M2/Macaulay2/packages/Visualize/ideal2d-example.html
+++ b/M2/Macaulay2/packages/Visualize/ideal2d-example.html
@@ -4,7 +4,7 @@
         <meta charset="utf-8">
         <title>VisIdeal2D</title>
        <!-- <script type="text/javascript" src="js/d3.v3.js"></script>-->
-        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
         <style>
@@ -219,10 +219,10 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
 <!--  <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 -->
--- a/M2/Macaulay2/packages/Visualize/ideal3d-example.html
+++ b/M2/Macaulay2/packages/Visualize/ideal3d-example.html
@@ -3,7 +3,7 @@
     <head>
       <meta charset="utf-8">
       <title>visIdeal</title> 
-      <link href="css/bootstrap.min.css" rel="stylesheet">
+      <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
       <link href="css/BootSideMenu.css" rel="stylesheet">
       <link href="css/nouislider.min.css" rel="stylesheet">
      <style>
@@ -90,11 +90,11 @@
 
 
     
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
-  <script src="js/bootstrap.min.js"></script>
-  <script src="js/three.min.js"></script> 
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/three/three.min.js"></script>
   <script src="js/underscore.js"></script> 
 		
 
--- a/M2/Macaulay2/packages/Visualize/poset-example.html
+++ b/M2/Macaulay2/packages/Visualize/poset-example.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Poset Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -77,11 +77,11 @@
         </div>
 
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visPoset.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 
--- a/M2/Macaulay2/packages/Visualize/simplicial-complex-example.html
+++ b/M2/Macaulay2/packages/Visualize/simplicial-complex-example.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Simplicial Complex Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -89,11 +89,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visSimplicialComplex2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
       
--- a/M2/Macaulay2/packages/Visualize/templates/visDigraph/visDigraph-template.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visDigraph/visDigraph-template.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Digraph Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -59,11 +59,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visDigraph2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 
--- a/M2/Macaulay2/packages/Visualize/templates/visGraph/visGraph-template.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visGraph/visGraph-template.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Graph Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -84,11 +84,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visGraph2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
       
--- a/M2/Macaulay2/packages/Visualize/templates/visIdeal/visIdeal2D.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visIdeal/visIdeal2D.html
@@ -4,7 +4,7 @@
         <meta charset="utf-8">
         <title>VisIdeal2D</title>
        <!-- <script type="text/javascript" src="js/d3.v3.js"></script>-->
-        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
         <style>
@@ -219,10 +219,10 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
 <!--  <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 -->
--- a/M2/Macaulay2/packages/Visualize/templates/visIdeal/visIdeal3D.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visIdeal/visIdeal3D.html
@@ -3,7 +3,7 @@
     <head>
       <meta charset="utf-8">
       <title>visIdeal</title> 
-      <link href="css/bootstrap.min.css" rel="stylesheet">
+      <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
       <link href="css/BootSideMenu.css" rel="stylesheet">
       <link href="css/nouislider.min.css" rel="stylesheet">
      <style>
@@ -75,11 +75,11 @@
 
 
     
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
-  <script src="js/bootstrap.min.js"></script>
-  <script src="js/three.min.js"></script> 
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/three/three.min.js"></script>
   <script src="js/underscore.js"></script> 
 		
 
--- a/M2/Macaulay2/packages/Visualize/templates/visPoset/visPoset-template.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visPoset/visPoset-template.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Poset Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -77,11 +77,11 @@
         </div>
 
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visPoset.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
 
--- a/M2/Macaulay2/packages/Visualize/templates/visSimplicialComplex/visSimplicialComplex2d-template.html
+++ b/M2/Macaulay2/packages/Visualize/templates/visSimplicialComplex/visSimplicialComplex2d-template.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Macaulay2: Simplicial Complex Visualization</title>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="/usr/share/javascript/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/BootSideMenu.css" rel="stylesheet">
     <link href="css/nouislider.min.css" rel="stylesheet">
     <link href="css/common.css" rel="stylesheet">
@@ -89,11 +89,11 @@
     <div id="canvasElement2d"></div>
   </div>
 
-  <script src="js/jquery-1.11.3.min.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.min.js"></script>
   <script src="js/BootSideMenu.js"></script>
-  <script src="js/d3.v3.min.js"></script>
+  <script src="/usr/share/javascript/d3/d3.min.js"></script>
   <script src="js/visSimplicialComplex2d.js"></script>
-  <script src="js/bootstrap.min.js"></script>
+  <script src="/usr/share/javascript/bootstrap/js/bootstrap.min.js"></script>
   <script src="js/clipboard.min.js"></script>
   <script src="js/nouislider.min.js"></script>
       
